INeedCoffee/calculate.rb

```diff
@@ -0,0 +1,25 @@
+require 'haversine'
+
+class Calculate
+  def self.DistanceBetweenCoffeeShopAndUser(coffeeShopName, coffeeShopLatitude,coffeeShopLongitude,userLatitude,userLongitude, distanceFromUserToCoffeeShop)
```

Comment:
Use naming conventions for ruby [guide](http://rubylearning.com/satishtalim/ruby_names.html)